CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+### v1.7.1 April 20, 2021
+
+* Fixed an issue where `onAttachedToEngine` was called twice on Android, causing it to crash when the plugin `com.google.gms.google-services` was added.
```

Comment:
This is not very accurate `when the plugin `com.google.gms.google-services` was added.`

Buy adding gms.google-services, the plugin works fine. It failed when firebase plugin is added and initialized in the application.

Suggest you update it as:
> Fixed an issue where `onAttachedToEngine` was called twice on Android, causing it to crash when the firebase plugin is added and initialized in the app.

Reply:
- [ ] update the change log message.